scripts/rebuild-deb.sh

```diff
@@ -82,6 +82,46 @@ ls -lh mina*.deb
 
 ##################################### END GENERATE KEYPAIR PACKAGE #######################################
 
+##################################### SNAPP TEST TXN #######################################
+
+mkdir -p "${BUILDDIR}/DEBIAN"
+cat << EOF > "${BUILDDIR}/DEBIAN/control"
+
+Package: mina-snapp-test-txn
+Version: ${MINA_DEB_VERSION}
+License: Apache-2.0
+Vendor: none
+Architecture: amd64
+Maintainer: o(1)Labs <build@o1labs.org>
+Installed-Size:
+Depends: ${SHARED_DEPS}
+Section: base
+Priority: optional
+Homepage: https://minaprotocol.com/
+Description: Utility to generate Snapp transactions in Mina GraphQL format
+ Built from ${GITHASH} by ${BUILD_URL}
+EOF
+
+echo "------------------------------------------------------------"
+echo "Control File:"
+cat "${BUILDDIR}/DEBIAN/control"
+
+# Binaries
+mkdir -p "${BUILDDIR}/usr/local/bin"
+cp ./default/src/app/snapp_test_transaction/snapp_test_transaction.exe "${BUILDDIR}/usr/local/bin/mina-snapp-test-transaction"
```

Comment:
Intuitively it feels to me that the name of the installed binary should match the name of the package.

Reply:
I'm going to change the package name to `mina-snapp-test-transaction`.